### ISSUE TITLE
Long Titles Overrun Boxes

### ISSUE THREAD

**Issue by drewofdoom** [OpenMusicKontrollers/patchmatrix#20]
![image](https://user-images.githubusercontent.com/2058589/112711441-d4236b00-8e9e-11eb-94ac-544ef87d3759.png)

Longer titles overrun the statically-set boxes. This problem is exacerbated when using Pipewire. BTW, Patchmatrix is the only patchbay I've used that properly works with Pipewire. Other patchbays will frequently show leftover lines when applications close.

 Auto-expanding boxes would resolve this issue and make Patchmatrix look even better (and it already looks damn good).

**Comment by ventosus** [comment]
thanks for reporting, which version is this?

    patchmatrix -v

**Comment by drewofdoom** [comment]
0.22.0 packaged in my COPR repo:
https://copr.fedorainfracloud.org/coprs/drewofdoom/fedora-proaudio/
https://github.com/drewofdoom/fedora-proaudio

```
➜ patchmatrix -v
patchmatrix 0.22.0
Copyright (c) 2016-2021 Hanspeter Portner (dev@open-music-kontrollers.ch)
Released under Artistic License 2.0 by Open Music Kontrollers
--------------------------------------------------------------------
This is free software: you can redistribute it and/or modify
it under the terms of the Artistic License 2.0 as published by
The Perl Foundation.

This source is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
Artistic License 2.0 for more details.

You should have received a copy of the Artistic License 2.0
along the source as a COPYING file. If not, obtain it from
http://www.perlfoundation.org/artistic_license_2_0.
```

**Comment by unsaph3** [comment]
Having the same issue in version 0.24, but in general this is a much better patchbay than the Cadence/Catia combo I had been using. Thanks for this

**Comment by ventosus** [comment]
fixed in 8acc1a8d4a83c0eaf873547c3b47c4915e9d6f8f
